docs/docs/flows.mdx

```diff
@@ -131,23 +131,23 @@ Steps can manifest in one of the following types:
 
 ### Starting a Flow
 
-How a flow can be triggered, depends on the first step in the flow. If the first
-step is a [user message step](./flows.mdx#user-message), the flow will be
-triggered by a user utterance that matches the intent or entities specified in
-the step.
+Flows can be triggered by other Rasa components, e.g. the 
+[LLM Command Generator](./llms/llm-command-generator.mdx). The description of each
+flow is very important for the LLM Command Generator, as it uses the description
+to determine which flow to trigger.
 
-If the first step is an [entry prompt step](./flows.mdx#entry-prompt), the flow
+An alternative way to trigger the flow directly through the Flow Policy is 
+the first step of the flow. If the first step is an 
+[entry prompt step](./flows.mdx#entry-prompt), the flow
```

Comment:
The first step isn't the way to trigger the flow. The entry prompts are. So I would say:

"An Alternative way to trigger flows is using entry prompts (link). Entry prompts trigger an LLM call with the given context and a yes/no question to determine whether to start the flow.